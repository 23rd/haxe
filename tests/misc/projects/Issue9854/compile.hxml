--main Main
--jvm bin/test.jar