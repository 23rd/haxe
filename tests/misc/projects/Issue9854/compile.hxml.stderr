Main.hx:13: characters 9-10 : Warning : String